lessons.py

```diff
@@ -57,27 +59,32 @@ def __init__(self):
 
             view_box = Gtk.EventBox()
             view_box.add(view)
-            view_box.modify_bg(Gtk.StateType.NORMAL, Gdk.color_parse(theme.WHITE))
+            view_box.modify_bg(Gtk.StateType.NORMAL,
+                               Gdk.color_parse(theme.WHITE))
             view_box.props.border_width = 10
 
             border_box = Gtk.EventBox()
             border_box.add(view_box)
-            border_box.modify_bg(Gtk.StateType.NORMAL, Gdk.color_parse(theme.WHITE))
+            border_box.modify_bg(Gtk.StateType.NORMAL,
+                                 Gdk.color_parse(theme.WHITE))
 
             scrolled_window = Gtk.ScrolledWindow()
-            scrolled_window.set_policy(Gtk.PolicyType.NEVER, Gtk.PolicyType.AUTOMATIC)
+            scrolled_window.set_policy(Gtk.PolicyType.NEVER,
+                                       Gtk.PolicyType.AUTOMATIC)
             scrolled_window.add_with_viewport(border_box)
 
             View.notebook.append_page(scrolled_window)
 
         self.show_all()
 
+
 _lessons_dir = os.path.join(get_bundle_path(), 'lessons')
 _lang = locale.getdefaultlocale()[0].split('_')[0]
 
 if not os.path.isdir(os.path.join(_lessons_dir, _lang)):
     logger.info('Cannot find lessons for language %s, thus use en' % _lang)
     _lang = 'en'
 
-for i, filename in enumerate(sorted(glob(os.path.join(_lessons_dir, _lang, '*')))):
+for i, filename in enumerate(sorted(glob(os.path.join(_lessons_dir,
```

Comment:
This is unreadable and sometimes some long lines are better left as long as they're readable as in this case there's quite the nested calls so not separating it right makes it unreadable.

I don't think this line should be split but if you wanted to do it, it's more readable having the `os.path` call on the new line so.

Reply:
Done.